test/handler.test.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable semi, no-unused-expressions */
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import sinon from 'sinon'
+import sinonChai from 'sinon-chai'
+import nock from 'nock'
+import Lambda from '../index.js'
+import ResHandler from '../src/responseHandlers.js'
+import Parser from '../src/epubParsers.js'
+import AccessibilityChecker from '../src/accessibilityCheck'
+import moment from 'moment'
+chai.should()
+chai.use(sinonChai)
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+describe('Handlers [index.js]', () => {
+  describe('exports.handler', () => {
+    let handlerStub, parseStub
+    beforeEach(() => {
+      handlerStub = sinon.stub(ResHandler, 'resultHandler')
+      parseStub = sinon.stub(Lambda, 'parseRecords')
+    })
+
+    afterEach(() => {
+      handlerStub.restore()
+      parseStub.restore()
+    })
+
+    it('should return 500 if no records in array', () => {
```

Comment:
Is an empty array indicating no records were found? It's a valid array object, but without a set of records to process to processing should end. I think we talked about this in stand up -- using a different status code like a 204 No Content response? That would indicate that the request was fulfilled and terminate the process.